### ISSUE TITLE
Unable to add rules in config

### ISSUE THREAD

**Issue by joacim-boive** [pa11y/pa11y-ci#58]
pa11y-ci 2.1.1

Using the below config.
```
{
    "defaults": {
        "standard": "WCAG2A",
        "rules": ["Principle1.Guideline1_1.1_1_1.H37"],
        "hideElements": "",
        "ignore": [],
        "includeWarnings": true,
        "timeout": 5000,
        "threshold": 0
    },
    "urls": [
        {
            "url": "https://www.ecster.se",
            "actions": [
            ]
        }]
}
```

I get:
```
 Error: Evaluation failed: Error: Principle1.Guideline1_1.1_1_1.H37 is not a valid WCAG 2.0 rule
   at configureHtmlCodeSniffer (<anonymous>:60:13)
   at runPa11y (<anonymous>:30:3)
```

As far as I understand this is according to the documentation:
https://github.com/pa11y/pa11y#rules-array

**Comment by rowanmanning** [comment]
Hi @joacim-boive, are you still having this issue? It may be that our documentation is out of date for the list of rules, or possibly HTML CodeSniffer has changed them.

Could you try running without the `rules` configuration and setting the standard to `WCAG2AAA`? Then try to find the rule that you want to include in the output?

**Comment by lsigism** [comment]
Hi @rowanmanning and @joacim-boive,

I am kind of having the same issue here =/ But checking the Pa11y HTMLCS.js (not sure if this is the right file to look at) I couldn't find the rule '_Principle1.Guideline1_1.1_1_1.H37_' mentioned by Joacim. I noticed that if we remove the ".H37" we are able to run without any errors (as the left bit 'Principle1.Guideline1_1.1_1_1' is listed in that file). 

And another thing, it seems that the documentation is not up to date (the rule's name used in a section). For example, accessing the documentation https://github.com/pa11y/pa11y#configuration in the section "_rules (array)_" we can see the following reference "_Principle1.Guideline1_3.1_3_1_AAA_". I tried to search for this reference here https://github.com/pa11y/pa11y/wiki/HTML-CodeSniffer-Rules and here http://squizlabs.github.io/HTML_CodeSniffer/Standards/WCAG2/ but I couldn't find it. Might be the case of just updating this reference to a valid rule. And about the rule _Principle1.Guideline1_1.1_1_1.H37_ it's being listed at those documents as a valid rule for all 3 levels of WCAG2.

**Comment by joacim-boive** [comment]
This solved my problem:
https://github.com/pa11y/pa11y-ci/issues/60
